### PR TITLE
Add hint to responsibles in booths

### LINKED ISSUE #1649: Hint in booths#form
The form in booths has a field called 'Responsibles'. That field lets you search for users and add them as a responsible for the booth you create. It searches from users that already exist, ie have an account in osem. 
We should add a hint to the responsibles field to communicate that.

### PR DESCRIPTION
**Pull request by rahul2240**
Fixes #1649 
booth-hint

### CODE REVIEW

File: app/views/admin/cfps/_form.html.haml [hunk 1]
```diff
@@ -8,5 +8,6 @@
       = f.input :start_date, as: :string, input_html: { id: 'registration-period-start-datepicker', start_date: @conference.start_date, end_date: @conference.end_date, readonly: 'readonly' }
       = f.input :end_date, as: :string, input_html: { id: 'registration-period-end-datepicker', readonly: 'readonly' }
       = f.input :cfp_type, as: :select, collection: (@cfp.new_record? ? @program.remaining_cfp_types : [@cfp.cfp_type] + @program.remaining_cfp_types).map {|type| ["#{type.capitalize}", type]}, include_blank: false, label: 'Type', input_html: { class: 'select-help-toggle' }
+      = f.description
```
Comment (differentreality) [inline app/views/admin/cfps/_form.html.haml]: What's this doing here?

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -166,7 +166,8 @@ def responsibles_selector_input(form)
     users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1].downcase }
     form.input :responsibles, as: :select,
                               collection: options_for_select(users.map {|user| ["#{user[1]} (#{user[2]}) #{user[3]}", user[0]]}, @booth.responsibles.map(&:id)),
-                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' }
+                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' },
+                              hint: 'This field lets you search for users and add them as a responsible for the booth you create. It searches from users that already exist, i.e have an account in osem.'
```
Comment (differentreality) [inline app/helpers/application_helper.rb]: Feel free to not copy paste my explanation from the issue and make this sentence shorter ;)

File: app/views/admin/cfps/_form.html.haml (hunk 1 above)
Comment (rahul2240) [reply app/views/admin/cfps/_form.html.haml]: oh i am sorry for this.

**Comment by rahul2240** [comment]
Please take a look now.

File: app/helpers/application_helper.rb [hunk 3]
```diff
@@ -166,7 +166,8 @@ def responsibles_selector_input(form)
     users = User.active.pluck(:id, :name, :username, :email).map { |user| [user[0], user[1].blank? ? user[2] : user[1], user[2], user[3]] }.sort_by { |user| user[1].downcase }
     form.input :responsibles, as: :select,
                               collection: options_for_select(users.map {|user| ["#{user[1]} (#{user[2]}) #{user[3]}", user[0]]}, @booth.responsibles.map(&:id)),
-                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' }
+                              include_blank: false, label: 'Responsibles', input_html: { class: 'select-help-toggle', multiple: 'true' },
+                              hint: 'Only existing users can be assigned responsible for the booth'
```
Comment (differentreality) [inline app/helpers/application_helper.rb]: 'can be assigned responsible' does not sound that correct.

What about this?
`The people responsible for the booth. You can only select existing users.`

Comment (rahul2240) [reply app/helpers/application_helper.rb]: ok i will add this

Comment (differentreality) [reply app/helpers/application_helper.rb]: Please do :)

Comment (rahul2240) [reply app/helpers/application_helper.rb]: Sorry for the delay.

**Comment by differentreality** [comment]
Nice work @rahul2240 :+1: Thank you :tada:

**Pull request merged**